docs/pip_transition.md

```diff
@@ -0,0 +1,76 @@
+## Transition from using pip packages
+
+This documents the proposed transition to no longer using pip packages (fully debianised system)
+
+### Rationale
+Currently we use PIP as well as debian packages to install some python modules. There are a few problems with this:
+  * We can't automate dependency tracking between pip and deb packages
+  * Not all pip packages are pinned, so upstream can push an update and break our customers updates without us knowing
+  * pip is a less robust package installer. In particular, it doesn't track what files it installed.
+  * We have some evidence that there may be a race condition which is causing python packages to be corrupted on install.
+  
+  
+Steps
+
+### 1.  Build python packages and add them to dr
+* Building:
+   * py2deb seems to work reasonably well. The only caveat is that it needs `pip-accel` to be manually installed first `pip install pip-accel` as otherwise you get a version of pip which pip-accel is not compatible with. Then do `pip install py2deb`.
+* Choice of packages and versions:
+  * It is not necessary to install pip once we transition.
+  * Some packages also already have raspbian versions. I propose to build new ones where the pip ones we currently install are newer, but not where they are older. There are only two where the raspbian version is newer (pam and futures). If this proves to be a problem, we can build the exact version and pin in the debian control file.
+  
```

Comment:
On reflection, I think we should just pin all the new deb versions to exactly the same as the current pip versions. We can always unpin them later, and there's going to be enough to QA in this release.